lib/Kconfig.debug

```diff
@@ -195,6 +195,21 @@ config DEBUG_BUGVERBOSE
 	  of the BUG call as well as the EIP and oops trace.  This aids
 	  debugging but costs about 70-100K of memory.
 
+config ERROR_REPORT_NOTIFY
+	bool "Expose memory error reports to the userspace"
+	depends on TRACING
+	default n
+	help
+	  When enabled, captures error reports from debugging tools like KFENCE
+	  or KASAN using console tracing, and provides two files:
+	   - /sys/kernel/error_report/last_report
```

Comment:
I think the formatting when using:
```
- ...
- ...
```
is bad when using it with Kconfig help text. Maybe just:
```
When enabled, captures error reports from debugging tools (such as KFENCE
and KASAN) via console tracing, and exposes reports in /sys/kernel/error_report:
the file last_report contains the last report, and report_count the total number of
reports.
<rest about user space programs>
```
I think mentioning PAGE_SIZE is irrelevant, because it's not anything that a user can act on, and most likely will forget anyway.

A better UX would be to copy '<trimmed>' at the end of reports that are actually trimmed.

Reply:
Hm, won't it be surprising to the users that we truncate the report without mentioning it at least somewhere?
Applied the rest of the text suggestions, will also implement report truncation.

Reply:
Maybe it's worth mentioning, but I think I just didn't like the calling out of "trimmed at PAGE_SIZE" of something that happens extremely rarely. The way it was put sounds like it's a common occurrence, and might be cause for criticism.

Maybe just: "<something about storing reports> (with maximum report length of PAGE_SIZE)".